Documentation/General/productVision.tex

```diff
@@ -0,0 +1,70 @@
+\documentclass[10pt,a4paper]{article}
+\usepackage[utf8]{inputenc}
+\usepackage{amsmath}
+\usepackage{amsfonts}
+\usepackage{amssymb}
+\begin{document}
+
+\title{Product Vision\\
+	\begin{small}
+	The why, who and what
+	\end{small}
+}
+\author{MoodCat.me}
+\maketitle
+
+\section{Why should we build MoodCat?}
+Listening to music is currently a solo activity.
+Various competitors provide a system that can offer users popular songs or songs that are similar to songs the user already listened to.
+These systems don't offer users songs they haven't heard yet and that might fit the user's current need.
+Since these systems provide solely songs that fit only the user and are very limited in varience, a better solution can be achieved.
+
+Secondly there are some competitors that offer users a room to socialize while experiencing songs selected by other users in the room.
+While this might offer the user unknown songs, the system is user-driven and does not offer the listeners that want to listen on their own.
+
+\section{What are the selling points of MoodCat?}
+MoodCat has several selling points that makes it unique in the current music market.
+
+\begin{enumerate}
+\item MoodCat is self-maintaining, which makes it possible to handle audience sizes from one to one thousand.
+This makes it possible to use the system no matter how many other users are currently online.
+
+\item MoodCat offers users both to experience solely or communicate with fellow music listeners.
+
+\item MoodCat does not suffer a lot of a cold start.
+The system is able to offer relevant songs using the specified current mood of the user.
```

Comment:
I think this is the most ideal scenario. But in fact, the cold start is one of the biggest challenges we have to deal with? I think this selling point should be: The system is corrected by users to prevent wrongly classified songs.